extensions/bloodsugar-converter/src/convert-from-mg-dl-to-mmol-l.ts

```diff
@@ -0,0 +1,13 @@
+import { LaunchProps, showToast } from "@raycast/api";
+import { showFailureToast } from "@raycast/utils";
+
+export default function Command(props: LaunchProps<{ arguments: Arguments.ConvertFromMgDlToMmolL }>) {
+  if (!Number.isInteger(props.arguments.value)) {
+    showFailureToast("Please enter a valid blood sugar value in mg/dL (whole number)");
+  }
+
+  const value = Number(props.arguments.value) / 18.018;
+  const rounded = Math.floor(value * 10) / 10;
+
+  showToast({ title: `${rounded} mmol/l` });
```

Comment:
(non-blocking) Worth investigating showing the toast as a HUD instead of a Toast. If you are reading from Clipboard or selection as suggested above, then definitely show the HUD or Toast as a HUD, instead of a Toast.

```suggestion
  closeMainWindow();
  showToast({ title: `${rounded} mmol/l` });
```